### PR TITLE
Table setup

### PR DESCRIPTION
**Pull request by Omwomo**
- Followed correct Gitflow.
- Created a database.
- Create a table named animals.
- Inserted data into the table.
- Queried data in the table following the query instructions provided.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE vet_clinic (
```
Comment (joskalenda) [inline schema.sql]: - [ ] Vet_clinic is meant to be the db name, not the table name, as per the requirements, you need to create a table name " `animals`, please update

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM vet_clinic WHERE name LIKE '%mon';
+SELECT name FROM vet_clinic WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name FROM vet_clinic WHERE neutured = true AND escape_attempts < 3;
+SELECT date_of_birth FROM vet_clinic WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM vet_clinic WHERE weight_kg > 10.5;
+SELECT * FROM vet_clinic WHERE neutured = true;
+SELECT * FROM vet_clinic WHERE name != 'Devimon';
```
Comment (joskalenda) [inline queries.sql]: - [ ] At this point, line 9 needs some assistance, you are asked to find animals that are not named `Gabumon` not Devimon.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Omwomo , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Good readme file
- [ ].. ✅ Good SQL queries
- [ ].... ✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @Omwomo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights
The animals is well created 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**